starter-code/server.js

```diff
@@ -1,18 +1,25 @@
 'use strict';
 
-// TODO: Initialize your project using NPM to create and populate a package.json file
+// DONE: Initialize your project using NPM to create and populate a package.json file
 
-// TODO: Require the Express package that you installed via NPM, and instantiate the app
+// DONE: Require the Express package that you installed via NPM, and instantiate the app
 // Remember to install express, and be sure that it's been added to your package.json as a dependency
 // There is also a package here called body-parser, which is required in for use in a new route.
 // Be sure to install that and save it as a dependency after you create your package.json.
+const express = require('express');
+const app = express();
 const bodyParser = require('body-parser').urlencoded({extended: true});
 const PORT = process.env.PORT || 3000;
 
-// TODO: Include all of the static resources as an argument to app.use()
+// DONE: Include all of the static resources as an argument to app.use()
+app.use(express.static('./public/'));
+
 
-// TODO: (STRETCH) Write a new route that will handle a request and send the new.html file back to the user
 
+// TODO: (STRETCH) Write a new route that will handle a request and send the new.html file back to the user
+app.get('/', function(request, response) {
```

Comment:
This is still just sending the file for the index. This is a stretch goal so no points off but you should in the future refactor as '/' is actually just the base route so it is just sending index.html and you want it to send new.html